src/Sarif/Schemata/Sarif.schema.json

```diff
@@ -372,59 +362,59 @@
 
         "deletedLength": 
         {
-          "description": "An optional integer specifying the number of bytes to delete, start at the byte offset specified by the offset property, measured from the beginning of the file.",
+          "description": "Specifies the number of bytes to delete, starting at the byte offset specified by offset, measured from the beginning of the file.",
           "type": "integer",
           "minimum": 1
         },
 
         "insertedBytes": 
         {
-          "description": "An optional string that specifies the byte sequence to be inserted at the byte offset specified by the offset property, measured from the beginning of the file.",
+          "description": "An optional string that specifies the byte sequence to be inserted at the byte offset specified by the 'offset' property, measured from the beginning of the file.",
           "type": "string"
         }
       },
-      "required": ["offset", "deletedLength", "insertedBytes"]
+      "required": ["offset"]
     },
     "result": 
     {
-      "description": "Represents one or more observations about an analysis target produced by a static analysis tool.",
+      "description": "A single result produced by a static analysis tool.",
       "additionalProperties": false,
       "type": "object",
       "properties": {
         "ruleId": 
         {
-          "description": "An opaque, stable identifier that uniquely identifies the specific rule associated with the result (e.g., CA2001).",
+          "description": "A stable, opaque identifier for the rule that was evaluated to produce the result.",
           "type": "string"
         },
 
         "kind": 
         {
-          "description": "A string specifying the kind of observation this result represents. This shall be one of the following: warning, error, pass, pending, note, notApplicable, internalError. If this member is not present, its implied value is 'warning'.",
+          "description": "The kind of observation this result represents. If this property is not present, its implied value is 'warning'.",
           "default": "warning",
-          "enum": [ "error","warning","pass","note","notApplicable","internalError","configurationError"]
+          "enum": [ "pass", "warning", "error", "notApplicable", "configurationError", "internalError", "note" ]
         },
 
         "fullMessage": 
         {
-          "description": "A string that comprehensively describes the result to the users.",
+          "description": "A string that describes the result.",
           "type": "string"
         },
 
         "shortMessage": 
         {
-          "description": "A short description that summarizes an issue to the user in one or two lines.",
+          "description": "A string that describes the result, displayed in user interface contexts where the available space is limited to a single line of text.",
```

Comment:
suggest 'displayed when visible space is limited to a single line of text'. concise.